data/src/main/java/camp/nextstep/edu/github/data/GithubRepositoryImpl.kt

```diff
@@ -0,0 +1,19 @@
+package camp.nextstep.edu.github.data
+
+import camp.nextstep.edu.github.domain.GithubRepoModel
+import camp.nextstep.edu.github.domain.GithubRepository
+
+internal class GithubRepositoryImpl(
+    private val githubService: GithubService = NetworkModule.githubService
+) : GithubRepository {
+
+    override suspend fun fetchGithubRepos(): Result<List<GithubRepoModel>> {
+        val response = githubService.fetchGithubRepos()
+        if (response.isSuccessful) {
+            val githubRepos = response.body()?.toModelList() ?: return Result.failure(Exception())
```

Comment:
```suggestion
        if (response.isSuccessful) {
            val githubRepos = response.body()?.toModelList().orEmpty()
```
이렇게 빈 리스트를 넘기는 것은 어떨까요?

Reply:
orEmpty()로 바로 빈리스트를 반환할 수 있군요~!!